### DISCUSSION TITLE
Important question please who know reply

### DISCUSSION THREAD

**Discussion by 3ala2khaddour** [sebanc/brunch#1010] [Q&A]
Please i want to ask can i install chrome os 89 on legacy bios by dual boot with windows 7 on separate local drive example : my windows 7 was installed on drive C and want to install chrome os on local drive D and both of this local drives C & D at the same Hardisk / if it possible please tell me how and thanks so much

**Comment by kerta1n** [comment]
Your explanation does not make too much sense, so I will explain here. You cannot dual-boot from the same disk on a BIOS system. You can, however boot from two separate disks. Meaning, if you try to install ChromeOS in the same disk that Windows resides, it is not possible. You can only have one drive. If you use two separate disks it may be possible. However, if you only have USB 2 and/or use a hard drive/flash drive, your install may not work correctly from being too slow.
